src/main/java/com/example/gradecalculator/web/controller/UserController.java

```diff
@@ -26,16 +30,22 @@ public class UserController {
 
     private final UserRepository userRepository;
     private final UserTypeRepository userTypeRepository;
+    private final AuthenticationManager authenticationManager;
```

Comment:
this field is never used

Reply:
yes, but its only for initializing the field; this should therefore also be removed

Reply:
you can quickly see this in the IDE as the field name is grey:
![image](https://github.com/IoT-Ausbildung/grade-calculator/assets/130460027/89e226a3-59b4-429a-894f-259bc166e218)


the message in the IDE is: "Private field 'authenticationManager' is assigned but never accessed"